.azure/pipelines/pr/Phonebook.Source.MockPeopleBackend.pr.yml

```diff
@@ -5,7 +5,7 @@ pr:
       - master
   paths:
     include:
-      - Phonebook.MockBackend/**
+      - Phonebook.Source.MockPeopleBackend/**
```

Comment:
please remove the wildcards. These paths working only without wildcards.

Reply:
well they are currently not working either way.